migrations/91_cbkLandSale.js

```diff
@@ -0,0 +1,23 @@
+const { deployProxy, upgradeProxy } = require('@openzeppelin/truffle-upgrades');
+
+const Blacksmith = artifacts.require("Blacksmith");
+const CBKLandSale = artifacts.require("CBKLandSale");
+const ExperimentToken = artifacts.require("ExperimentToken");
+const CBKLand = artifacts.require("CBKLand");
+module.exports = async function (deployer, network, accounts) {
+
+  const cbkLand = await deployProxy(CBKLand, [], { deployer });
+  const cbkLandSale = await deployProxy(CBKLandSale, [cbkLand.address], { deployer });
+ 
+ 
+  const blacksmith = await upgradeProxy(Blacksmith.address, Blacksmith, { deployer });
+  const experimentToken = await ExperimentToken.deployed();
```

Comment:
correct me if I'm wrong but AFAIK we never use experiment token on any mainnets, we'd have to hardcode the address of king for each network in this migration to pass its reference instead.
Some other migrations use these with an if statement checking for dev network, so we can keep it operational on local

Reply:
I was going to ask to have conditional checks to know what address to put but that is not practical and would mean a migration for every new token. So I will add functions to add currencies.